frontend_tests/node_tests/emoji_picker.js

```diff
@@ -125,3 +132,61 @@ set_global('popovers', {
     assert($(".emoji-popover-category-tabs").visible());
     assert(!$(".emoji-search-results-container").visible());
 }());
+
+(function test_toggle_emoji_popover() {
+    var element = $('fake-add-emoji');
```

Comment:
Actually the thing is that $.create() doesn't allow us to create same object twice but I was doing that here. We need to remove the assertion from the zjquery for that and I guess it was there for intention (I probably forgot about it again)